### DISCUSSION TITLE
What is the right macOS update process with Legacy Video patch installed?

### DISCUSSION THREAD

**Discussion by mrdc** [chris1111/Legacy-Video-patch#23] [Q&A]
Hello,

I’m using Legacy Video patch for 10.15.7 and trying to figure out what’s the correct update process for macOS: can I just update via System Preferences as usual and then re-apply Legacy Video patch?
New update is out and it’s not clear how to perform it without killing the system.

**Comment by chris1111** [comment]
No because you have to deal with apple Update and maybe system is unbootable in the first Catalina its easy to update then reapply legacy Video but now its an adventure

**Comment by mrdc** [comment]
The latest 10.15.7 update kills the system: ui freezes at desktop etc.
Looks like Legacy video patch should be reverted before the update.

**Reply by chris1111** [reply]
> The latest 10.15.7 update kills the system: ui freezes at desktop etc.
> Looks like Legacy video patch should be reverted before the update.

its impossible to revert
